src/ds/logger.h

```diff
@@ -23,6 +30,79 @@ namespace logger
     MAX_LOG_LEVEL
   };
 
+  /** Print a demangled stack backtrace of the caller function to FILE* out. */
+  static inline void print_stacktrace(
```

Comment:
I think we should move this to another file as it provides more complex capabilities than what the logger does. Also, we should replace `std::cout` with `LOG_INFO` so that is uses the ring buffer for logging.

Reply:
Agreed, inside the enclave we could use the OE facility, and outside something like https://github.com/bombela/backward-cpp, which is available header only.